src/unix/udp.c

```diff
@@ -490,7 +501,7 @@ static int uv__udp_set_membership4(uv_udp_t* handle,
   int optname;
   int err;
 
-  memset(&mreq, 0, sizeof mreq);
+  memset(&mreq, 0, sizeof(mreq));
```

Comment:
Can you revert all of the unrelated style changes.

Reply:
Fixed. BTW, why the style is inconsistent in file (and project) scope?

Reply:
> why the style is inconsistent in file (and project) scope?

Because we haven't agreed on and enforced automated linting. So, some stuff gets by the human eye test.